cpp/include/kvikio/shim/cuda.hpp

```diff
@@ -15,8 +15,7 @@
  */
 #pragma once
 
-#include <cuda.h>
-
+#include <kvikio/shim/cuda_h_wrapper.hpp>
```

Comment:
*suggestion/nit*: Shall we call this `kvikio/shim/cuda_h.hpp` ? Or maybe `shim/cuda.hpp` ?

Reply:
We already have `kvikio/shim/cuda.hpp` and `kvikio/shim/cufile.hpp`. 
Should we rename the wrappers to:
```kvikio/shim/cuda_h.hpp
kvikio/shim/cufile_h.hpp` ?
Or maybe

Reply:
I think I prefer `XXX_stub.hpp` over `XXX_h.hpp`. Though neither of them are fully satisfactory. I guess my only quibble with `wrapper` is that it suggests something that the header file is not doing: in the case where the real header _is_ available, it just `#include`s that and no more.